pages/attributes.vue

```diff
@@ -0,0 +1,342 @@
+    >Attributes stand for properties you can add on OpenGeode objects, typically meshes. 
+    These properties are not directly stored on meshes, so you can imagine add Attributes
+    on other objects. </p>
+
+    <p
+      :class="paragraph"
+    >Attributes are managed by an 
+    <span v-html="attributeManager" />.
+    Meshes have manager defined on their vertices, edges, facets, and so on.
+    Updating the mesh (e.g. by adding or removing vertices) automatically updates attached
+    <span v-html="attributeManager" />. </p>
+
+    <p
+      :class="paragraph"
+    >But you can create an
+    <span v-html="attributeManager" />
+    that is not attached to a mesh.</p>
+    
+    <h2 :class="section">Create an Attribute</h2>
+
+    <p
+      :class="paragraph"
+    >First of all, let's get an  
+    <span v-html="attributeManager" />
+    by the dedicated mesh methods or by creating it:</p>
+
+    <!-- eslint-disable -->
+    <code class="cpp primary--text">
+      AttributeManager manager;
+      manager.resize( 10 );
+      auto attribute_ptr = manager.find_or_create< VariableAttribute, double >(
+        "first_attribute", 12.3 );
+      manager.resize( 6 );
+    </code>
+    <!-- eslint-enable -->
+
+    <p
+      :class="paragraph"
+    >The first line of the above code creates a default  
+    <span v-html="attributeManager" />.</p>
+
+    <p
+      :class="paragraph"
+    >The second line resizes the created
+    <span v-html="attributeManager" />.
+    It means that all Attributes associated to this 
+    <span v-html="attributeManager" /> will have exactly 10 values.
+    </p>
+
+    <p
+      :class="paragraph"
+    >Notice that this method can be called before and after 
+    the creation of Attributes associated to the 
+    <span v-html="attributeManager" /> 
+    (as done at the fourth line of the above code). 
+    </p>
+
+    <p
+      :class="paragraph"
+    >As in standard library, there is also a method to reserve Attributes capacities. 
+    </p>
+
+    <p
+      :class="paragraph"
+    >Now, take a look at the third line of the above code. 
+    This line creates an attribute using the method <code>find_or_create</code>.
+    Four elements are needed to define the Attribute to create:
+    </p>
+
+    <ul :class="paragraph">
+      <li>
+        An <strong>attribute name</strong> (here <code>first_attribute</code>): 
+        the first parameter of the method <code>find_or_create</code>;
+      </li>
+      <li>
+        The <strong>type</strong> of Attribute storage (here a 
+        <span v-html="variableAttribute" />): 
+        the first template parameter of the method (more details below);
+      </li>
+      <li>
+        The <strong>type</strong> of Attribute values (here <code>double</code>): 
+        the second template parameter of the method;
+      </li>
+      <li>
+        The <strong>default value</strong> (here <code>12.3</code>): 
+        the second parameter of the method.
+      </li>
```

Comment:
```suggestion
      <li>
        The <strong>type</strong> of Attribute storage (here a 
        <span v-html="variableAttribute" />): 
        the first template parameter of the method (more details below);
      </li>
      <li>
        The <strong>type</strong> of Attribute values (here <code>double</code>): 
        the second template parameter of the method;
      </li>
      <li>
        An <strong>attribute name</strong> (here <code>first_attribute</code>): 
        the first parameter of the method <code>find_or_create</code>;
      </li>
      <li>
        The <strong>default value</strong> (here <code>12.3</code>): 
        the second parameter of the method.
      </li>
```
Fifi (same order than the method)